packages/core/bootstrap/src/lib/cache-warmer/metrics.ts

```diff
@@ -0,0 +1,11 @@
+import * as client from 'prom-client'
+
+export const cache_warmer_count = new client.Gauge({
```

Comment:
I would merge both of these into one metric, with a `isBatched` boolean style label to differentiate between the two.